### ISSUE TITLE
Support for dragging in macOS Mojave floating screen shots

### ISSUE THREAD

**Issue by gklka** [sindresorhus/caprine#646]
If you take a screen shot in macOS Mojave, the shot will float in the screen edge for a few seconds. You can take this image and drag where you like, this way the file won’t be saved, so you don’t have to clean up after you sent it.

Caprine (similarly to Slack) doesn’t receive this type of image, only the files the OS saves after a few seconds.

It is very frustrating that I have to wait a while before I can send a screen shot.

**Comment by sindresorhus** [comment]
Yeah, I have been frustrated by that too, especially in Slack... I'm pretty sure this is a limitation in Electron, as you can reproduce the same by trying to drag the floating screenshot into Slack or Messenger in Chrome.

**Comment by St1c** [comment]
no, in Chatty (which is also probably electron app) it works correctly.

**Comment by whitecrownclown** [comment]
@St1c [Chatty](https://github.com/chatty/chatty) is written in `Java`. I had a look over this and you can't drop the files in `Chrome` or `Electron`. You just won't receive an event. Trying to drop into `Safari` works btw.

[This](https://bugs.chromium.org/p/chromium/issues/detail?id=915410&q=component%3ABlink%3EDataTransfer%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified) chromium issue is somewhat related. I think a separate issue should be opened.
